test/TokenRequestTest.js

```diff
@@ -110,6 +110,19 @@ contract('TokenRequest', ([rootAccount, ...accounts]) => {
         'TOKEN_REQUEST_ADDRESS_NOT_CONTRACT'
       )
     })
+    it('reverts when acceptedDepositTokens length is more than max tokens', async () => {
```

Comment:
Would change to something like `reverts when passed token list with more tokens than the max accepted`